react/features/toolbox/components/Toolbox.native.js

```diff
@@ -119,15 +120,25 @@ class Toolbox extends Component {
      * @returns {ReactElement}
      */
     render() {
+        if (!this.props._visible) {
+            return null;
+        }
+
         return (
             <Container
-                style = { styles.toolbarContainer }
-                visible = { this.props._visible }>
+                style = {
+                    isNarrowAspectRatio(this)
+                        ? styles.toolbarContainerNarrow
+                        : styles.toolbarContainerWide } >
                 {
-                    this._renderPrimaryToolbar()
+                    isNarrowAspectRatio(this)
+                        ? this._renderSecondaryToolbar()
+                        : this._renderPrimaryToolbar()
                 }
                 {
-                    this._renderSecondaryToolbar()
+                    isNarrowAspectRatio(this)
```

Comment:
this code looks weird when reading, is there no other way to write it a bit come compact? maybe a helper method called `_renderToolbars` which takes the orientation into consideration?

Reply:
can do - yes